bitcoin/src/crypto/sighash.rs

```diff
@@ -994,21 +994,22 @@ impl<R: Borrow<Transaction>> SighashCache<R> {
             match sighash {
                 EcdsaSighashType::All => {
                     self_.output.consensus_encode(writer)?;
-                },
+                }
                 EcdsaSighashType::Single => {
                     // sign all outputs up to and including this one, but erase
                     // all of them except for this one
                     let count = input_index.min(self_.output.len() - 1);
                     writer.emit_compact_size(count + 1)?;
                     for _ in 0..count {
                         // consensus encoding of the "NULL txout" - max amount, empty script_pubkey
-                        writer.write_all(&[0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0x00])?;
+                        writer
+                            .write_all(&[0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0x00])?;
```

Comment:
This looks like a bug, unless this length is controlled by some percentage of `max_width`. I tried messing with the `rustfmt` config but just ended up with a ton of warnings.

FTR the width is 99 characters before this change.

Reply:
Acknowledged. I'm still gonna merge this unless you think we should fight rustfmt over it.

Reply:
Yeah merge please.